### DISCUSSION TITLE
support option counting like click

### DISCUSSION THREAD

**Discussion by xiaopeng163** [fastapi/typer#727] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the Typer documentation, with the integrated search.
- [X] I already searched in Google "How to X in Typer" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to Typer but to [Click](https://github.com/pallets/click).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
@click.command()
@click.option('-v', '--verbose', count=True)
def log(verbose):
    click.echo(f"Verbosity: {verbose}")
```

### Description

are we going to support option counting just like click? thanks.

### Operating System

Linux, Windows, macOS

### Typer Version

0.9.0

### Python Version

3.11.4

**Comment by apinkney97** [comment, marked as answer]
FYI this seems to be already supported:

https://typer.tiangolo.com/tutorial/parameter-types/number/#counter-cli-options
